daemons/compute/api/datamovement.proto

```diff
@@ -98,28 +98,31 @@ message DataMovementCreateRequest {
     // If True, the data movement command runs `/bin/true` rather than perform actual data movement
     bool dryrun = 4;
 
+    // Extra options to pass to `mpirun` if present in the Data Movement command.
+    string mpirunOptions = 5;
+
     // Extra options to pass to `dcp` if present in the Data Movement command.
-    string dcpOptions = 5;
+    string dcpOptions = 6;
```

Comment:
I would think that this is frowned upon by protobuf people?
Better to add the new field at the end.

Reply:
Lots of little guidelines regarding these numbers: https://protobuf.dev/programming-guides/proto3/